zanata-model/src/main/java/org/zanata/hibernate/search/ContainingWorkspaceBridge.java

```diff
@@ -42,8 +42,6 @@ public void set(String name, Object value, Document luceneDocument,
 
         addStringField(IndexFieldLabels.PROJECT_FIELD, project.getSlug(),
                 luceneDocument, luceneOptions);
-        addStringField(IndexFieldLabels.ITERATION_FIELD, iteration.getSlug(),
-                luceneDocument, luceneOptions);
```

Comment:
We don't use iteration slug in any lucene query. We used to use it in project wide search but then we switched to use HQL instead. Do you know any where else it is used?